service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/scheduling/EhrExtractTimeoutScheduler.java

```diff
@@ -22,9 +22,9 @@
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class EhrExtractTimeoutScheduler {
 
-    private static final String UNEXPECTED_CONDITION_ERROR_CODE = "99";
-    private static final String UNEXPECTED_CONDITION_ERROR_MESSAGE
-                                            = String.format("No acknowledgement has been received within %s days", 8);
+    private static final String ERROR_CODE = "99";
+    private static final String ERROR_MESSAGE = "No acknowledgement has been received within ACK timeout limit";
```

Comment:
Just as a side note again, I think having some documentation around what these `ERROR_CODE`'s are (assuming that there is none) could be beneficial. Furthermore, perhaps extracting these into some enumeration?

Reply:
good suggestion, I will introduce the Enum and think about the docs.